docs/program-setup/runtime/Execution.md

```diff
@@ -1,7 +1,34 @@
+When writing an Mbed OS program, in whichever development tool you use, you may need to:
+
+* Set up a connection to your board so you can flash and debug your program.
+    * You won't usually need a driver for Mbed enabled boards
+    * Mbed CLI has a built-in serial terminal to display output from your application
```

Comment:
```suggestion
    * Mbed CLI has a built-in serial terminal to display output from your program.
```

Reply:
Just so it matches "flash and debug your program" - you can also change both to application, which is far more common in Mbed OS ("program" is being dragged over from Mbed Studio)